modules/packages/manifests/p7zip.pp

```diff
@@ -0,0 +1,11 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class packages::p7zip {
```

Comment:
We no longer need 7zip, so we can drop this :)

Reply:
Yeah, I'd removed the installation but had left the packages manifest in case it was useful elsewhere.